packages/flame_svg/lib/svg_component.dart

```diff
@@ -4,9 +4,12 @@ import 'package:flame/components.dart';
 
 import './svg.dart';
 
+/// Wraps a [Svg] into a Flame component.
 class SvgComponent extends PositionComponent {
+  /// [Svg] instance
```

Comment:
These type of docs are the ones I'm afraid we will get too many of when we get in to the Flame codebase.

Reply:
Yep, but the pros of the other docs outweight the cons of these useless comments IMO :D

Reply:
You're probably right!

Reply:
Another possibility is to make it private

Reply:
how about `The wrapped instance of [Svg].` (given that the class doc says it wraps Svg into a component)

Reply:
```suggestion
  /// The wrapped instance of [Svg].
```